### ISSUE TITLE
Uncaught exception 'PDFlibException' with message 'Bad floating point number for PDF'

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jkoo7yeha/dompdf#242]
```
What steps will reproduce the problem?
1.Create a HTML page
2.use pdflib for rendering
3.render it 

What is the expected output? What do you see instead?

new pdf file should be generate but i got this error

Fatal error: Uncaught exception 'PDFlibException' with message 'Bad floating 
point number for PDF' in 
/data/www/3dgolf.i-tee.nl/www/icms/dompdf/include/pdflib_adapter.cls.php:680 
Stack trace: #0 /dompdf/include/pdflib_adapter.cls.php(680): 
PDFlib->fit_textline('this is test', 8, 767.04, 'rotate=0 wordsp...') #1 
/dompdf/include/text_renderer.cls.php(84): PDFLib_Adapter->text(8, 12, 'this is 
test', '/data/www/3dgol...', 12, Array, 0) #2 
/dompdf/include/renderer.cls.php(180): 
Text_Renderer->render(Object(Text_Frame_Decorator)) #3 
/dompdf/include/renderer.cls.php(96): Renderer->_render_frame('text', 
Object(Text_Frame_Decorator)) #4 /dompdf/include/renderer.cls.php(132): 
Renderer->render(Object(Text_Frame_Decorator)) #5 
/dompdf/include/renderer.cls.php(132): 
Renderer->render(Object(Block_Frame_Decorator)) #6 /data/www/3dgolf.i in 
/dompdf/include/pdflib_adapter.cls.php on line 680

What version of the product are you using? On what operating system?

@version 0.5.1 on Linux

Please provide any additional information below.

```

Original issue reported on code.google.com by `wian...@gmail.com` on 8 Mar 2011 at 4:26

**Comment by GoogleCodeExporter** [comment]
```
We had issues with "localized" floating point number with commas instead of 
points, but they were fixed in dompdf 0.6
```

Original comment by `fabien.menager` on 28 Apr 2012 at 11:15
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
Original comment by `eclecticgeek` on 30 May 2013 at 5:15
- Added labels: **Restrict-AddIssueComment-Commit**
